docs/guides/server/caching.adoc

```diff
@@ -349,6 +349,25 @@ To set up for IPv6 only and have {project_name} pick the bind address automatica
 export JAVA_OPTS_APPEND="-Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true"
 ----
 
+== Running behind a firewall or reverse proxy
```

Comment:
if you have a firewall, it won't be able to connect except if you open the JGroups port. 
The `tunnel` stack is the way to go for a closed firewall environment, but they require an accessible gossip router instance.

Reply:
Reading that, I think the heading needs improvement. Also "reverse proxy" is misleading as most would assume a HTTP reverse proxy, which this is not. 

Maybe change the heading to "Running with port forwarding" ?

Reply:
proxy or port forward looks good to me. I'm unsure about which one is more correct. 

Here is the property description if it helps
https://github.com/belaban/JGroups/blob/master/src/org/jgroups/protocols/TP.java#L74-L77